apps/entry/management/commands/fix_ahhs_data.py

```diff
@@ -0,0 +1,75 @@
+import csv
+import os
+import logging
+from decimal import Decimal
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from apps.entry.models import Figure
+from utils.common import round_half_up
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    help = "Fix AHHS household size data."
+
+    def add_arguments(self, parser):
+        parser.add_argument('csv_file_path', type=str, help="Path to the CSV file containing the data.")
+
+    # FIXME Use Bulk Manager
+    def patch_figure_household_from_csv(self, file_path):
+        change_in_repoted_value_figures = []
+        change_in_unit_figures = []
+        figure_id_not_found = []
+
+        with open(file_path, 'r') as file:
+            reader = csv.DictReader(file)
+            for row in reader:
+                fig_id = row['figure_id']
+                figure = Figure.objects.get(id=fig_id)
+                if not figure:
+                    figure_id_not_found.append(fig_id)
+                    print("No figure found for ids:", fig_id)
+                    continue
+
+                if figure.unit != Figure.UNIT.HOUSEHOLD:
+                    change_in_unit_figures.append(figure.id)
+                    print("Figure unit is not Household")
+                    continue
+
+                if figure.reported != int(row['figure_reported_value']):
+                    change_in_repoted_value_figures.append(figure.id)
+                    print("Change in reported value")
+                    continue
+
+                if figure.household_size == row['figure_household_size']:
+                    change_in_repoted_value_figures.append(figure.id)
+                    print("No change in household size")
+                    continue
+
+                figure.household_size = row.get('figure_household_size')
+                figure.total_figures = round_half_up(
+                    int(row['figure_reported_value']) * Decimal(str(row['figure_household_size']))
+                )
+                figure.save()
+
+        print("Figure ids not found:", figure_id_not_found)
+        print("Figure unit not household:", change_in_unit_figures)
+        print("Change in reported value figure ids:", change_in_repoted_value_figures)
```

Comment:
Let's use stdout

```suggestion
        self.stdout.write(f'Figure ids not found: {figure_id_not_found}')
        self.stdout.write(f'Figure unit not household: {change_in_unit_figures}')
        self.stdout.write(f'Change in reported value figure ids: {change_in_repoted_value_figures}')
```

Also, do we need to show changed count or figure ids?